db/model/token.js

```diff
@@ -109,6 +109,36 @@ module.exports = function token(seq, dataTypes) {
     }));
   };
 
+  /**
+   * Deletes tokens unused since the provided threshold. Deletes both revoked
+   * AND un-revoked tokens. Does nothing if the "since" threshold does not
+   * resolve to a negative number of milliseconds.
+   *
+   * @param {String|Integer} since - a time offset e.g. "-1d" for one day ago,
+   *  or numeric value -360000 for one minute ago.
+   * @returns {Promise<Integer>} number of records deleted
+   */
+  Token.deleteUnused = function (since) {
+    try {
+      // eslint-disable-next-line no-magic-numbers
+      const millis = ms(since) || 0;
+      if (millis < 0) { // eslint-disable-line no-magic-numbers
+        return Token.destroy({
+          where: {
+            lastUsed: {
+              [Op.lt]: new Date(new Date().getTime() + ms(since)),
```

Comment:
I think this would be more readable with Date.now() so we're not doing new Date() twice.